node/server.js

```diff
@@ -0,0 +1,26 @@
+// Remember, in a real application, you should ensure your user is authenticated and authorized to view the chart before you return the signed URL.
+
+// Replace these constants with the correct values for your Charts instance
+const CHARTS_EMBEDDING_BASE_URL = '~REPLACE~CHARTS_EMBEDDING_BASE_URL'  // Replace with the base URL to your Charts instance, e.g. https://charts.mongodb.com/charts-foo-abcde
+const CHARTS_TENANT_ID = '~REPLACE~CHARTS_TENANT_ID' // Replace with your Charts Tenant ID from the Embed Chart snippet
+const EMBEDDING_SIGNING_KEY = '~REPLACE~EMBEDDING_SIGNING_KEY' // Replace with the Embedding Signing Key generated by your Charts admin
+const EXPIRY_TIME_SECONDS = 300 // Set to your preferred expiry period
+
+const express = require('express');
+const crypto = require('crypto');
+const app = express();
+const port = 4567;

```

Comment:
Minor feedback: The default port for express applications is `3000`. One could argue that changing this port is a little more secure, but given it is a demo, as a user of express I would goto `localhost:3000` without giving it too much thought, so it might be worth changing. 0/10 FG™️.